src/components/menus/BottomMenu.module.scss

```diff
@@ -57,46 +93,42 @@
 
   @media (min-width: $desktop) {
     text-align: left;
-    justify-content: flex-start;
-    align-items: flex-start;
   }
 }
 
-.MenuLinkTitle {
+.MenuLinkTitle,
+.legalTitle {
   font-family: $font-header;
+  color: $color-black;
   padding-bottom: 20px;
   padding-top: 20px;
-  font-size: 20px;
-  font-weight: bold;
+  font-size: 24px;
+  font-style: italic;
+  font-weight: 400;
 }
 
-.ContactColumn {
-  border-top: 1px solid $color-lightorange;
-  border-bottom: 1px solid $color-lightorange;
-  text-align: center;
-  width: 100%;
-  max-width: width(12);
-  margin-top: 30px;
-  margin-bottom: 20px;
-  padding-bottom: 30px;
+.MenuLinkTitle::before {
+  content: '\2756';
+  margin-right: 5px;
+  color: $color-sage;
+  font-size: 18px;
+}
 
+.contacts {
+  text-align: center;
   .info {
     p {
       font-size: 0.78em;
-
-      &.address {
-        padding-top: 8px;
-      }
-    }  
+      line-height: 1.3rem;
+      font-weight: 600;
+      letter-spacing: 0.1rem;
```

Comment:
Line-height and letter spacing should generally always be done via em since if the font size changes, the letter spacing and line height should change with the font changes. This will help when the user goes to up their browser font size or the component moves. REM is great if you really need to set things based on the page or reset them but you'll want to carry on with Em from there. If this is confusing, feel free to reach out and we can chat through it.